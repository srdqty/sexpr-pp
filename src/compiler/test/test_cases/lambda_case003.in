(lambda (x y z x) 1 2 3)
