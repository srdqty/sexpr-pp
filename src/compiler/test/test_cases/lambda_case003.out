stdin:1:16: lambda: duplicate parameter name: x
